site/docs/reference/Connectors/materialization-connectors/PostgreSQL.md

```diff
@@ -77,4 +77,10 @@ You can find the values for `forwardHost` and `forwardPort` in the following loc
 * Amazon RDS: `forwardHost` as Endpoint; `forwardPort` as Port.
 * Google Cloud SQL: `forwardHost` as Private IP Address; `forwardPort` is always `5432`. You may need to [configure private IP](https://cloud.google.com/sql/docs/postgres/configure-private-ip) on your database.
 * Azure Database: `forwardHost` as Server Name; `forwardPort` under Connection Strings (usually `5432`).
-:::
\ No newline at end of file
+:::
+
+## Notes
```

Comment:
I'd suggest just making "reserved words" the H2 and take this out.

Reply:
Done 👍🏽